### ISSUE TITLE
[iOS] How to update Pjsip in existing project?

### ISSUE THREAD

**Issue by Rushang08** [pjsip/pjproject#2315]
So I have really 5 years old pjsip project and now I wanted to update pjsip latest version because of the opus codec.

**Comment by trengginas** [comment]
If you are using 2.x and wanted to update to the latest (2.10) then you need to check the **backward incompatibility** text included on the release notes of each release prior to 2.10.
e.g:
If you are using 2.8 and want to update to 2.10, check release notes for [2.9](https://trac.pjsip.org/repos/milestone/release-2.9) and [2.10](https://trac.pjsip.org/repos/milestone/release-2.10).

If you don't use any API listed, then you shouldn't need to change any API you called. Note that some platform (e.g: iOS) might introduce new requirements which is not handled by the library.

**Comment by Rushang08** [comment]
I tried to update the library but having unknown crash on pjsua_call_get_info.

**Comment by Rushang08** [comment]
`/* Callback called by the library when call's state has changed */
static void on_call_state(pjsua_call_id call_id, pjsip_event *e)
{
    
    pjsua_call_info ci;
    PJ_UNUSED_ARG(e);

    pjsua_call_get_info(call_id, &ci);

    PJ_LOG(1,(THIS_FILE, "Call %d state=%.*s", call_id,
              (int)ci.state_text.slen, ci.state_text.ptr));
    
    if (ci.state == PJSIP_INV_STATE_DISCONNECTED)
    {
        sip_ring_stop([[CallManager sharedCallManager] pjsipAppConfig]);
        call_deinit_tonegen(call_id);
    }
    else if (ci.state == PJSIP_INV_STATE_EARLY)
    {
        int code;
        pj_str_t reason;
        pjsip_msg *msg;

        pj_assert(e->type == PJSIP_EVENT_TSX_STATE);

        msg = (e->body.tsx_state.type == PJSIP_EVENT_RX_MSG ?
               e->body.tsx_state.src.rdata->msg_info.msg :
               e->body.tsx_state.src.tdata->msg);

        code = msg->line.status.code;
        reason = msg->line.status.reason;
//        msg->body == NULL
        if (ci.role == PJSIP_ROLE_UAS && (code == 180 || code == 183) &&
            ci.media_status == PJSUA_CALL_MEDIA_NONE)
        {
            sip_ringback_start([[CallManager sharedCallManager] pjsipAppConfig]);
        }
    }

    if (ci.state != PJSIP_INV_STATE_NULL)
    {
        [[CallManager sharedCallManager] handleCallStateWithCall_id:call_id AndCallInfo:&ci];

    }
}`


This is my code and if I
 comment ` pjsua_call_get_info(call_id, &ci);` then it's crash on **handleCallStateWithCall_id** 
Basically wan't able to retrive info from **pjsua_call_info** and so that **ci->remote_contact.ptr**
Please refer below screenshot.

![Screenshot 2020-03-05 at 4 51 50 PM](https://user-images.githubusercontent.com/8275040/75977068-9ead8780-5f01-11ea-99ec-e2965b275ec6.png)
